documentation/globalflags.md

```diff
@@ -0,0 +1,36 @@
+# Global Flags
+
+These flags are global. They affect all subcommands.  
+
+```text
+   --debug, -v        Enable detailed logging (default: false)
+   --allow-fetch      Enable JS fetch(), dangerous on untrusted code! (default: false)
+   --disableordering  Disables update reordering (default: false)
+   --no-colors        Disable colors (default: false)
+   --help, -h         show help
+```
+
+They must appear before the subcommand.
+
+```
+RIGHT:
```

Comment:
I suggest removing 'right' and 'wrong' from the shell command blocks.

**Right**:

```shell
dnscontrol --no-colors preview
```